editor/effects.js

```diff
@@ -37,11 +37,15 @@ import {
 	isEditedPostDirty,
 	isEditedPostNew,
 	isEditedPostSaveable,
+	getMetaBoxes,
 } from './selectors';
 
 const SAVE_POST_NOTICE_ID = 'SAVE_POST_NOTICE_ID';
 const TRASH_POST_NOTICE_ID = 'TRASH_POST_NOTICE_ID';
 
+// Hold jquery.ready until the metaboxes load
+jQuery.holdReady( true );
```

Comment:
Should we guard it with check for `jQuery` existence to make sure it works in test env, too?

Reply:
Okey, you mocked it. It's okayish :)

Reply:
The `wp-editor` script does not define a dependency on jQuery. We can't otherwise assume this global exists.

https://github.com/WordPress/gutenberg/blob/4da6c0f317a148ad87ce630c327abb4e3fb498d3/lib/client-assets.php#L603-L609

Reply:
Is it possible to move this into the effect for one of the initialization actions? Not sure I like the side-effect we're introducing to the top-level scope here. Maybe better to at least consolidate this somewhere, as in `editor/index.js`?

As I'm reviewing this, I'm still working through understanding why we need this; can you clarify more in a comment why we need to hold?

Reply:
>I'm still working through understanding why we need this; can you clarify more in a comment why we need to hold?

I believe we need to currently hold due to waiting on the iframes to load, so that any scripts can properly add their listeners when jQuery is ready.

I don't think that is a long term need, now that we are on post.php we can render the metaboxes server side, then take those dom nodes and bring them into React.